backend/sources/api/helm/BUILD

```diff
@@ -0,0 +1,28 @@
+load("@rules_pkg//:pkg.bzl", "pkg_tar")
+load("@com_github_airyhq_bazel_tools//helm:helm.bzl", "helm_template_test")
+load("//tools/build:helm.bzl", "helm_push")
+
+filegroup(
+    name = "files",
+    srcs = glob(
+        ["**/*"],
+        exclude = ["BUILD"],
+    ),
+    visibility = ["//visibility:public"],
+)
+
+pkg_tar(
+    name = "package",
+    srcs = [":files"],
+    extension = "tgz",
+    strip_prefix = "./",
+)
+
+helm_template_test(
+    name = "template",
+    chart = ":package",
+)
+
+helm_push(
+    chart = ":package",
+)
```

Comment:
nitpick: These lines seem to be the same everywhere. If we add a `helm_package` macro in the tools directory we can avoid rewriting them everywhere.

Reply:
Got it :+1:. Will do it that way.